src/status_im2/contexts/onboarding/identifiers/style.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im2.contexts.onboarding.identifiers.style)
+
+(def page-container
+  {:flex 1})
+
+(def content-container
+  {:position :absolute
+   :width    "100%"
+   :top      160
+   :bottom   0
+   :left     0})
+
+(def card-style
+  {:margin-horizontal 20
+   :margin-bottom     :auto})
```

Comment:
File should end with a new line, right?

Reply:
@ajayesivan, this one ends with a newline *character*. I know, it's confusing, but having a newline *character* is different than having an extra empty newline.

This SO is very useful https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

> Because that’s how the POSIX standard defines a line https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206:
>
>    3.206 Line
>       A sequence of zero or more non- <newline> characters plus a terminating <newline> character.
>
> Therefore, lines not ending in a newline character aren't considered actual lines. That's why some programs have problems processing the last line of a file if it isn't newline terminated.